ui/src/components/policy/ManagedPolicy.js

```diff
@@ -27,7 +29,11 @@ const ManagedPolicy = () => {
 
   const [availableManagedPolicies, setAvailableManagedPolicies] = useState([]);
   const [selected, setSelected] = useState(null);
-
+  const [
+    attachedManagedPolicyDetails,
+    setAttachedManagedPolicyDetails,
+  ] = useState(null);
+  const editorRef = useRef();
```

Comment:
Nice that you are using useRef for referencing editor. does the react-monaco-editor use React Ref natively?